src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4decoder/common/ControllingMetricGroup.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4decoder.common;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
```

Comment:
Please update the new format of this javadocs.

for intellij:
#if (${PACKAGE_NAME} && ${PACKAGE_NAME} != "")package ${PACKAGE_NAME};#end
#parse("File Header.java")
/**
 * ${NAME} 
 * 
 * @author Ivan / Symphony Dev Team<br>
 *		   Created on ${DATE}
 * @since 1.0.0
 */
public class ${NAME} {
}